src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -109,6 +110,11 @@ function CreatePipelineRun() {
     );
   }
 
+  function getYAMLMode() {
```

Comment:
maybe `isYAMLMode` and return boolean?

current usage is a little odd with https://github.com/tektoncd/dashboard/pull/2575/files#diff-a76903232a0bbce3293cb530f3aac343e2b5d6dda21048219bb2467ce9b5027dR469

```js
const yamlMode = getYAMLMode();
if (yamlMode && yamlMode === 'yaml') {
```

instead could just have:
```js
if (isYAMLMode()) {
```